tsconfig.json

```diff
@@ -3,6 +3,10 @@
   "compilerOptions": {
     "rootDir": ".",
     "outDir": "build",
+    "lib": [
+      "es2016",
+      "esnext.bigint"
```

Comment:
Interesting :) bigint only appears to be the latest versions of 10 and up:
https://node.green/#ESNEXT-candidate--stage-3--BigInt-basic-functionality

Is this something we can natively use if we want to support 8?

Reply:
We don't really do anything with BigInts, but I thought it would be nice if we offered them as a return type via separate method.
```js
const date = new PreciseDate();
date.getFullTime(); // BigInt
date.getFullTimeString(); // string version
```

Reply:
As long as it won't blow people on 8 up, I'm coo